docs/getting-started/provision-infrastructure.md

```diff
@@ -1,258 +1,333 @@
 ---
 title: Provision Infrastructure
 toc: true
-weight: 3
+weight: 2
 indent: true
 ---
 
-# Provision Infrastructure
-
-Crossplane allows you to provision infrastructure anywhere using the Kubernetes
-API. Once you have [installed a provider] and [configured your credentials], you
-can create any infrastructure currently supported by the provider. Let's start
-by provisioning a database on your provider of choice.
-
-Each provider below offers their own flavor of a managed database. When you
-install a provider it extends Crossplane by adding support for several "managed
-resources". A managed resource is a cluster-scoped Kubernetes custom resource
-that represents an infrastructure object, such as a database instance.
+# Compose Infrastructure
+
+In the [last section] we installed packages that extend Kubernetes with
```

Comment:
These two paragraphs don't match the current structure. For example, we did not actually `used them to provision resources on a cloud`. Maybe we can move `Configuration`-related parts to the former doc and keep a short description of what claim means and how it's similar to PVC, then point the full composition doc.

Reply:
I think just eliminating `used them to provision resources on a cloud` would be sufficient to make this work, but I am open to moving these to another doc if desired.

Reply:
How do we feel about just moving this entire intro section to: https://github.com/crossplane/crossplane/pull/2166/files#diff-fd6af16be5d1d97ecd20ef62757aed21bc0ddc1a30a4b320e2d4b343af453610R204

(which I think is roughly what @muvaf is suggesting here)

Reply:
I tend to agree that explanation of what XRDs and Compositions are belongs close to where we install them for the first time; I don't think it ruins the magic to give a little detail with regard to what exactly the Configurations will be installing.

I'd introduce the concept of a claim here, close to where we install them, if we can factor that out without a huge rewrite. So I think that would mean pretty much everything except the "XRs are always cluster scoped - ..." paragraph moves back to the Install & Configure docs.